src/components/inputField/PasswordField/PasswordField.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from 'react';
+import styles from './PasswordField.module.scss';
+import hide from '../../../assets/hidePasswordIcon.png';
+import show from '../../../assets/showPasswordIcon.png'; 
+
+/***
+ * Parameters for the password field
+ * header: Required. String of the header for the textbox. Example: "Enter Password"
+ * setInputValue: Required. Variable to set the input value of the password
+ */
+interface PasswordFieldProps {
+    header: string;
+    setInputValue: React.Dispatch<React.SetStateAction<string>>;
```

Comment:
This should never be the type of callback functions, they just need to be a function type.
So this would be something like setInputValue: (value: string) => void.
The reasoning for this is having the SetState type means this component expects the parent to use it a certain way and it shouldn't care about that it should be a dumb component.